kubejs/data/ptd/advancements/root.json

```diff
@@ -21,4 +20,4 @@
     "requirements": [ [ 
         "0"
     ] ]
-}
```

Comment:
This is the best change here

Reply:
Something to do with editing it in the browser